nixos/maintainers/scripts/ec2/amazon-image.nix

```diff
@@ -70,6 +76,9 @@ in
     };
   };
 
+  config.virtualisation.diskSize = lib.mkDefault (3 * 1024);
```

Comment:
This could potentially break someones config because previously this option was set via `mkOptionDefault` i.e. 1500. If someone set this value via `mkDefault` in their own config, it would've overriden the value. Now it doesn't anymore.

Maybe it's better to set this to 1500 -1 so that it overrides the default value from the option but can still be overriden via `mkDefault`. I don't have a super strong opinion though.

Reply:
I know numbers are kind ridiculous, but they're spaced apart to provide ample opportunity to wiggle more things in as needed. I'd go for like 1490, just for the unlikely case that something else needs to be in between.

Reply:
Good catch, thank you! Just changed it to use a priority of 1490 for all of the renamed options.